### ISSUE TITLE
Syntax error in the most recent version.

### ISSUE THREAD

**Issue by tanema** [activemerchant/payment_icons#206]
```
SassC::SyntaxError (Error: error in C function image-url: non-conforming drawing primitive definition `c.707' @ error/draw.c/TracePath/5624
        on line 19:23 of ../../.gem/ruby/2.6.5/gems/payment_icons-1.3.2/app/assets/stylesheets/payment_icons/_payment_icons.scss, in function `image-url`
        from line 19:23 of ../../.gem/ruby/2.6.5/gems/payment_icons-1.3.2/app/assets/stylesheets/payment_icons/_payment_icons.scss
        from line 1:9 of app/assets/stylesheets/admin/modules/_payment_icons.scss
        from line 66:9 of app/assets/stylesheets/admin/style.scss
>>     background-image: image-url("payment_icons/#{$svg-name}.png");

   ----------------------^
):

app/assets/config/manifest.js:79
```

**Comment by tauthomas01** [comment]
Hi @tanema , thank you for reporting this. The only PR that would be the cause is the change I did: https://github.com/activemerchant/payment_icons/pull/204

That fix was to resolve the CI build failing. I couldn't think of another idea.
I'll have to dig in further.

**Comment by tauthomas01** [comment]
cc. @rafaelfranca @AnotherJoSmith @unixcharles

**Comment by rafaelfranca** [comment]
The 1.3.1 version is not tagged, but that was the release that causes this problem. https://github.com/activemerchant/payment_icons/compare/v1.3.0...v1.3.2. The commit is https://github.com/activemerchant/payment_icons/commit/a2cf71c0d26c39cce9c3ab781c82e9e675c250bd. It adds a svg with some data that can't be converted to png in some versions of imagemagik.

You can see the `c.707` inside that svg code.

**Comment by tauthomas01** [comment]
Thank you for providing more context, @rafaelfranca .

The icon was submitted by @Korri . Would you have time to investigate this? Maybe the way how the icon was generated isn't compatible with Imagemagik.

**Comment by Korri** [comment]
👋my knowledge about SVG and graphics in general is really limited, but I used Affinity Designer to design and export it.

**Comment by tauthomas01** [comment]
@Korri I made an update for the icon. Could you check it? https://github.com/activemerchant/payment_icons/pull/207

**Comment by SebastianSzturo** [comment]
This should be fixed now in v1.3.3.
